packages/suite/src/views/settings/SettingsDevice/SettingsDevice.tsx

```diff
@@ -120,6 +124,7 @@ export const SettingsDevice = () => {
                     ) : (
                         <>
                             <BackupRecoverySeed isDeviceLocked={isDeviceLocked} />
+                            {supportsShamir && <MultiShareBackup />}
```

Comment:
The condition is redundant. Please either move the condition from `MultiShareBackup` to a util and use it here, or remove the condition from here.